config/forms/2022_2023.json

```diff
@@ -7470,7 +7470,7 @@
                 "household_charge": {
                   "check_answer_label": "Does the household pay rent or charges?",
                   "header": "Does the household pay rent or other charges for the accommodation?",
-                  "hint_text": "",
+                  "hint_text": "If rent is charged on the property then answer Yes to this question, even if the tenants do not pay it themselves.",
```

Comment:
Yes should be in quotes? "Yes"

Reply:
@kosiakkatrina I spoke to Jemma, and we agreed it wasn't needed.